config/config-example.js

```diff
@@ -157,6 +157,7 @@ exports.loguserstats = 1000 * 60 * 10; // 10 minutes
 // amount of traffic (i.e. hundreds of concurrent battles).
 exports.validatorprocesses = 1;
 exports.simulatorprocesses = 1;
+exports.verifierprocesses = 1;
```

Comment:
So... no processes and no config? 1 process by default in config? I want to be sure before I do anything